.github/actions/publish-image/action.yaml

```diff
@@ -45,17 +48,19 @@ outputs:
 runs:
   using: composite
   steps:
+    - uses: imjasonh/setup-crane@00c9e93efa4e1138c9a7a5c594acd6c75a2fbf0c # v0.3.0
     - shell: bash
       id: docker-publish
       env:
         REGISTRY: ${{ inputs.registry }}
         REPO: ${{ inputs.repository }}
         REGISTRY_PASSWORD: ${{ inputs.registry-password }}
         COSIGN_REPOSITORY: ${{ inputs.sbom-repository }}
+        IMAGE_TAG: ${{ inputs.image-tag }}
       run: |
         set -e
         make ${{ inputs.makefile-target }}
-        echo "digest=$(docker inspect --format='{{index .RepoDigests 0}}' ghcr.io/${{ github.repository }}:${{ github.sha }})" >> $GITHUB_OUTPUT
+        echo "digest=ghcr.io/${{ github.repository }}@$(crane digest ghcr.io/${{ github.repository }}:${{ inputs.image-tag }})" >> $GITHUB_OUTPUT
```

Comment:
Is this the digest only or the full image?

Reply:
its the full image

Reply:
Would it work with the existing workflow https://github.com/nirmata/kyverno-notation-aws/pull/153/files#diff-84bc20808405b0dac7891b6b85bf24263c21cba412b4d0f52c470f9e8e33274cR82?